src/test/java/utils/Waiter.java

```diff
@@ -0,0 +1,15 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import static driver.BasicDriver.webDriver;
+
+public class Waiter {
```

Comment:
Waiter is someone in restaurant . Be more specific

Reply:
;)Changed to ExplicitWait